package.json

```diff
@@ -34,7 +34,7 @@
     "sandbox:build": "cd sandbox && npm run build",
     "dev": "npm-run-all --parallel build:watch sandbox",
     "checkbundlesize": "npm run build:prod:standalone:minified && bundlesize",
-    "mockserver": "mockserver -serverPort 1080 -jvmOptions \"-Dmockserver.enableCORSForAllResponses=true\""
```

Comment:
I'm not sure how you were getting this to work. The mockserver docs seem to show the library API but no CLI.

Reply:
I must have installed mockserver with `brew`.  Thanks for fixing this!